beauty-consulting-app/app/Pages/Messaging/ChatPage.js

```diff
@@ -1,17 +1,215 @@
-import React from "react";
-import { View, Text } from "react-native";
+import React, { useContext, useState, useEffect, useRef } from "react";
```

Comment:
please add a back button, no way to get back to conversations page from chat page